lib/tdf3/src/utils/zip-writer.ts

```diff
@@ -100,7 +100,7 @@ export class ZipWriter {
   zip64: boolean;
   constructor() {
     // Enable zip64
-    this.zip64 = false;
+    this.zip64 = true;
```

Comment:
Can you pull the zip64 changes out into a separate PR for ease of rollback in case they don't work right?

Reply:
Yes let me yank those out good idea! Although Misha tested them against the c++ core it's better safe than sorry 👍